clients/segments/segments.go

```diff
@@ -85,46 +77,46 @@ func modifyBody(source []byte) ([]byte, error) {
 	return body, nil
 }
 
-// Get gets a complete configuration of segment with an ID
-func (c Client) Get(ctx context.Context, id string) (Response, error) {
-	resp, err := c.client.Get(ctx, id, rest.RequestOptions{CustomShouldRetryFunc: rest.RetryIfTooManyRequests})
-	if err != nil {
-		return Response{}, fmt.Errorf(errMsgWithId, "get", id, err)
+func (c Client) Get(ctx context.Context, id string) (api.Response, error) {
+	if id == "" {
+		return api.Response{}, fmt.Errorf(errMsgWithId, "get", id, errors.New(`argument "id" is empty`))
```

Comment:
Correct me if I'm wrong, but we decided to go with single quotes, right (also for other usages in here)?

Reply:
True, I just copied as it was. Can do it in the follow-up by simplifying and improving the code. Here, the goal was to only remove the `api/clients`.